/**
 * 
 */
/**
 * 
 */
module Exercicios_throw_throws_custem_exceptions_2 {
}